go.mod

```diff
@@ -67,7 +67,6 @@ require (
 	github.com/vektah/gqlparser/v2 v2.0.1
 	go.opencensus.io v0.22.2
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	golang.org/x/mod v0.3.0 // indirect
```

Comment:
unintentional I assume?

Reply:
yes, unintentional. I will address this and make sure it matches the master branch.